media/css/firefox/new/desktop/download.scss

```diff
@@ -936,7 +936,7 @@ button.mzp-c-cta-link {
         @include text-body-sm;
         @include bidi((
             (border-left, 1px solid $color-marketing-gray-20, border-right, 0),
-            (text-align, center, center), // needs higher specificity for RTL
```

Comment:
Caught when I ran `npm format` not related to this work. I guess I could pull it out if you want.